packages/rum-core/src/domain/assembly.ts

```diff
@@ -68,6 +65,33 @@ export function startRumAssembly(
   buildCommonContext: () => CommonContext,
   reportError: (error: RawError) => void
 ) {
+  MODIFIABLE_FIELD_PATHS_BY_EVENT = {
+    [RumEventType.VIEW]: VIEW_MODIFIABLE_FIELD_PATHS,
+    [RumEventType.ERROR]: assign(
+      {
+        'error.message': 'string',
+        'error.stack': 'string',
+        'error.resource.url': 'string',
```

Comment:
From @yannickadam, see if we can retrieve some existing types rather than re-declaring it here

Reply:
Actually, it does not seem so since elsewhere it is only typings and here we need a value.

Reply:
👍